ci-operator/step-registry/osd-ccs/cluster/deprovision/osd-ccs-cluster-deprovision-commands.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+# Log in
+OCM_VERSION=$(ocm version)
+OCM_TOKEN=$(cat "${CLUSTER_PROFILE_DIR}/ocm-token")
+if [[ ! -z "${OCM_TOKEN}" ]]; then
+  echo "Logging into ${OCM_LOGIN_ENV} with offline token using ocm cli ${OCM_VERSION}"
+  ocm login --url "${OCM_LOGIN_ENV}" --token "${OCM_TOKEN}"
+  if [ $? -ne 0 ]; then
```

Comment:
`set -o errexit` indicates that the script will exit if any non-0 code is returned. So `if [ $? -ne 0 ]; then` is not required.

Reply:
Actually, we do not need this validation, the `ocm login` will validate everything. I will remove this `if` block.